schemas/iso19139/src/main/plugin/iso19139/index-fields/index-subtemplate.xsl

```diff
@@ -164,5 +161,15 @@
     <xsl:call-template name="subtemplate-common-fields"/>
   </xsl:template>
 
+
+  <!-- Indexing DQ report -->
+  <xsl:template mode="index" match="gmd:DQ_DomainConsistency[count(ancestor::node()) =  1]">
+    <xsl:variable name="title"
+                  select="gmd:result/gmd:DQ_ConformanceResult/gmd:specification/gmd:CI_Citation/gmd:title/(gco:CharacterString|gmx:Anchor)"/>
+    <resourceTitle><xsl:value-of select="$title"/></resourceTitle>
```

Comment:
Maybe better to do like for contact because we query on resourceTitleObject (cf. https://github.com/geonetwork/core-geonetwork/blob/main/web-ui/src/main/resources/catalog/js/CatController.js#L1092)
```suggestion
    <resourceTitleObject type="object">{
      "default": "<xsl:value-of select="gn-fn-index:json-escape($title)"/>"
      }</resourceTitleObject>
```
Depends a bit on how you use this type of subtemplates in the editor.